### ISSUE TITLE
Explain `caliper` scale better in the docs.

### ISSUE THREAD

**Issue by markmfredrickson** [markmfredrickson/optmatch#109]
A user pointed out that the default value of `width = 1` in the caliper function is indicative of an SD scale on the variable used to create the distance matrix. This default should probably be removed to avoid this suggestion, as caliper has to work on the distance scale. A note should be included pointing users to the `match_on` function, which works on the SD scale by default for GLMS and can be made to work on the SD scale pretty easily for the numeric method. Likewise, documentation for `match_on` can make this point more clear as well.

**Comment by benthestatistician** [comment]
I agree with both proposals (remove default caliper width, add
documentation cross-ref to `match_on`, which some users won't be explicitly
invoking).

On Tue, Dec 15, 2015 at 12:21 PM, Mark Fredrickson <notifications@github.com

**Comment by benthestatistician** [comment]
Related comment from a user:

> when you are using a caliper in combination with matching within strata (like I am doing with state), I don’t think the documentation specifies whether pooled SDs are calculated separately within each stratum or just calculated once on the dataset as a whole.

**Comment by markmfredrickson** [comment]
I added a paragraph about caliper working on the distance scale. It addresses the second question (about stratified matching) in that it explicitly states that SDs are not computed/used. 

It is a more subtle point that using the `within` argument to `match_on` to create a stratified match also will not use within strata SDs. I've added some notes to the glm and formula methods on this point.
